### ISSUE TITLE
ask for more documentation

### ISSUE THREAD

**Issue by redguardtoo** [org-mime/org-mime#2]
Hi, @acowley,
In ffbf3e3, you said,
    Support mime encoding current subtree
    
    This doesn't require that the user explicitly narrow to a subtree, and
    it adds support for export options set as properties on the subtree.
    
    The latter is fairly crucial as it lets one do things such as suppress
    table of contents generation on a subtree without doing so for the
    entire document.

What does the last paragraph mean? Can you show me some sample? I'd like to update manual about this feature.

Besides, can you test the latest org-mime.el with your `export options`?

**Comment by acowley** [comment]
It seems to no longer work. The issue for me was that I would get a Table of Contents in the HTML (and perhaps author and date, I don't fully recall), and so I made it so that `subtree-htmlize` would respect `EXPORT_OPTIONS` if defined in a `PROPERTIES` drawer.

The way I checked this just now was to pick a random Org file and run `org-mime-org-subtree-htmlize` on a heading with subheadings. The HTML output includes a ToC. If I then set

```
  :PROPERTIES:
  :EXPORT_OPTIONS: toc:nil title:nil author:nil num:nil
  :END:
```

underneath that heading and run the `subtree-htmlize` function again, I still get a ToC in the HTML output.

**Comment by redguardtoo** [comment]
fixed in 91d8f48 can set org export options (Chen Bin)

may you should set `EXPORT_TITLE` to nil because here is code quoted from `org-export--get-subtree-options`,
```lisp
(when (setq prop (or (org-entry-get (point) "EXPORT_TITLE")
			  (progn (looking-at org-complex-heading-regexp)
				 (org-match-string-no-properties 4))))
       (setq plist
	     (plist-put
	      plist :title
	      (org-element-parse-secondary-string
	       prop (org-element-restriction 'keyword)))))
```

**Comment by redguardtoo** [comment]
I also added `org-mime-export-options`, so you don't setup export options per buffer or per subtree.
